### DISCUSSION TITLE
Please refund Firefox License.

### DISCUSSION THREAD

**Discussion by chechoora** [aclap-dev/video-downloadhelper#1000] [License & Purchase requests]
### What's your issue?

I have licenses for Chrome and Edge. I specifically bought a Firefox license to download YouTube videos. However, it fails every time to the point that my connection goes down. I liked the Chrome and Edge versions, but I would like a refund for Firefox.
Date: 2 Jul, 2024

### The 10 first digits (only 10) of your license number or email address

07896c8292

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
